verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -127,6 +133,50 @@ export class AuthController {
             },
         );
 
+        this.router.post(
```

Comment:
Signing up as an email user fails for me when I try locally, and the error message isn't very helpful: it just says "Rejected". If I inspect the network request in the browser I see that posting to this endpoint got a 504 Gateway timeout error: what is it trying to talk to upstream when a user signs up?

Reply:
I couldn't reproduce this error. Both sign in and sign up work as expected on my local machine. Were you testing that while running the app with `dev/run_stack.sh` script?

Reply:
No, neither can I now, might have been a configuration problem on my side. OK, this is good to go!